src/OrchardCore/OrchardCore.Abstractions/Modules/Builder/OrchardCoreBuilder.cs

```diff
@@ -56,6 +57,26 @@ public OrchardCoreBuilder ConfigureServices(Action<IServiceCollection> configure
             return ConfigureServices((s, sp) => configure(s), order);
         }
 
+        /// <summary>
+        /// This async method gets called for each tenant. Use this method to initialize container services.
+        /// </summary>
+        /// <param name="initializeAsync">The async task to execute when initializing the services for a tenant..</param>
```

Comment:
```suggestion
        /// <param name="initializeAsync">The task to execute when initializing tenant services.</param>
```

Reply:
Idem, for example just before we already have this existing comment.

        /// <param name="configure">The action to execute when configuring the services for a tenant.</param>

I'm open to any suggestion but maybe a separate PR to review all of these comments.

At least there is an extra dot to remove.